assets/newspack-ui/scss/variables/_colors.scss

```diff
@@ -0,0 +1,48 @@
+:root {
+	// Newspack Blues
+	--newspack-ui-color-primary-000: #f5fdff;
+	--newspack-ui-color-primary-050: #d6f6ff;
+	--newspack-ui-color-primary-100: #b4e4ff;
+	--newspack-ui-color-primary-200: #93ccfd;
+	--newspack-ui-color-primary-300: #72affb;
+	--newspack-ui-color-primary-400: #528dfc;
+	--newspack-ui-color-primary-500: #36f;
+	--newspack-ui-color-primary-600: #2240d5;
+	--newspack-ui-color-primary-700: #1522af;
+	--newspack-ui-color-primary-800: #0b0b8d;
+	--newspack-ui-color-primary-900: #0d046e;
+	--newspack-ui-color-primary-950: #0e0052;
```

Comment:
Not for this PR, but we should tweak this to incorporate a dynamic primary color generated in the backend. The backend logic should use HSL to determine if the publisher's primary color hue is between 50 and 300 (not red). If it's red-ish, it should fall back to a dark grey.

I wonder how to merge that with the proposed grades. If we are to implement different gradings for the publisher's color, I'd propose we do it with opacity rather than dynamically calculating lightness. Still, I'm not a fan of messing with the publisher's brand color in that way...

Correct me if I'm wrong, but I think we only implement the different grades in the checkbox/radio input lists. I _think_ these should always be blue, but we should double-check. If that's the case, we don't need to worry about it.

Reply:
> we should tweak this to incorporate a dynamic primary color generated in the backend

Yes! 

> I wonder how to merge that with the proposed grades.

I copied these colours over as-is [from the _colors.scss file used for the back-end components](https://github.com/Automattic/newspack-plugin/blob/master/assets/shared/scss/_colors.scss) when I was under the impression a lot more of the back-end would be using Newspack blue. So I think we can probably pare this down to whatever we're actually using. 

> Correct me if I'm wrong, but I think we only implement the different grades in the checkbox/radio input lists. I think these should always be blue, but we should double-check. If that's the case, we don't need to worry about it.

This is my impression now, too! I'll follow up and confirm just in case 🙂

Reply:
I ended up removing the blue variables in [3bb8809](https://github.com/Automattic/newspack-plugin/pull/2810/commits/3bb8809a9c40527e54dcf1cb709812bef3a9f051) -- should I make a separate task for the custom colour stuff?